### DISCUSSION TITLE
Use output 'text' readonly as bytes data

### DISCUSSION THREAD

**Discussion by tridattran** [wierkstudio/ciphereditor#36] [Help]
Hi, not sure where to ask questions, so please forgive me if this is not the place and let me know where.

I'm trying to get the base64 encoded values of a HMAC output. 
The link from 'readonly' only text output of the HMAC function cannot be linked to as bytes to the binary to text encoding. (Red arrow in screen shot).

Is there a way of doing this? Am I using the wrong functions? At the moment the workaround is to copy and paste the data manually.

- [editor link](https://ciphereditor.com/share#blueprint=eyJ0eXBlIjoiYmx1ZXByaW50IiwicHJvZ3JhbSI6eyJ0eXBlIjoicHJvZ3JhbSIsIm9mZnNldCI6eyJ4IjotODUsInkiOjgyNX0sImZyYW1lIjp7IngiOi0xNjAsInkiOi05Niwid2lkdGgiOjMyMCwiaGVpZ2h0IjoxOTJ9LCJjaGlsZHJlbiI6W3sidHlwZSI6Im9wZXJhdGlvbiIsIm5hbWUiOiJAY2lwaGVyZWRpdG9yL2V4dGVuc2lvbi1oYXNoL2htYWMiLCJleHRlbnNpb25VcmwiOiJodHRwczovL2Nkbi5jaXBoZXJlZGl0b3IuY29tL2V4dGVuc2lvbnMvQGNpcGhlcmVkaXRvci9leHRlbnNpb24taGFzaC8xLjAuMC1hbHBoYS4xL2V4dGVuc2lvbi5qcyIsInByaW9yaXR5Q29udHJvbE5hbWVzIjpbImtleSIsImhhc2giLCJtZXNzYWdlIiwiYWxnb3JpdGhtIl0sImZyYW1lIjp7IngiOi02NTcsInkiOjM0NCwid2lkdGgiOjMyMCwiaGVpZ2h0Ijo1NzR9LCJjb250cm9scyI6eyJtZXNzYWdlIjp7InZhbHVlIjoiaGVsbG8td29ybGQiLCJ2aXNpYmlsaXR5IjoiZXhwYW5kZWQifSwia2V5Ijp7InZhbHVlIjoic29tZS1yYW5kb20ta2V5IiwidmlzaWJpbGl0eSI6ImV4cGFuZGVkIn0sImFsZ29yaXRobSI6eyJ2YWx1ZSI6InNoYTI1NiIsInZpc2liaWxpdHkiOiJleHBhbmRlZCJ9LCJoYXNoIjp7InZhbHVlIjoiOTJkOWQyYjVjZmE1NDU4ZjI5NTU5ZjIxYzBmMTdkNTk4YTMxMzI5MTc0ODViYjllMmFiODc1M2RjZmUzYmRhNiIsInZpc2liaWxpdHkiOiJleHBhbmRlZCJ9fX0seyJ0eXBlIjoib3BlcmF0aW9uIiwibmFtZSI6IkBjaXBoZXJlZGl0b3IvZXh0ZW5zaW9uLWVzc2VudGlhbHMvYmluYXJ5LXRvLXRleHQiLCJleHRlbnNpb25VcmwiOiJodHRwczovL2Nkbi5jaXBoZXJlZGl0b3IuY29tL2V4dGVuc2lvbnMvQGNpcGhlcmVkaXRvci9leHRlbnNpb24tZXNzZW50aWFscy8xLjAuMC1hbHBoYS4xL2V4dGVuc2lvbi5qcyIsInByaW9yaXR5Q29udHJvbE5hbWVzIjpbImRhdGEiLCJlbmNvZGVkRGF0YSIsImFscGhhYmV0IiwicGFkZGluZyJdLCJmcmFtZSI6eyJ4IjotMjEyLCJ5IjozNDQsIndpZHRoIjozMjAsImhlaWdodCI6NTE4fSwiY29udHJvbHMiOnsiZGF0YSI6eyJ2YWx1ZSI6eyJ0eXBlIjoiYnl0ZXMiLCJkYXRhIjoia3RuU3RjK2xSWThwVlo4aHdQRjlXWW94TXBGMGhidWVLcmgxUGMvanZhWT0ifSwidmlzaWJpbGl0eSI6ImV4cGFuZGVkIn0sImFscGhhYmV0Ijp7InZpc2liaWxpdHkiOiJleHBhbmRlZCJ9LCJlbmNvZGVkRGF0YSI6eyJ2YWx1ZSI6Imt0blN0YytsUlk4cFZaOGh3UEY5V1lveE1wRjBoYnVlS3JoMVBjL2p2YVk9IiwidmlzaWJpbGl0eSI6ImV4cGFuZGVkIn19fV19fQ)

<img width="779" alt="image" src="https://github.com/user-attachments/assets/4062a4ee-76ae-4cbc-8013-fbfc3e775a52">

**Comment by Jordy3D** [comment]
Is this not what you're trying to do?
![chrome_ayCkd6MdZ2](https://github.com/user-attachments/assets/3d791266-6cf3-4242-a89e-429592ee5609)

**Reply by tridattran** [reply]
Hi, yes, but notice the Data has 'Text', that needs to be 'Bytes'. So I want to treat the 'Text' output as bytes

Red arrow pointing to what I want to be bytes. 
<img width="651" alt="image" src="https://github.com/user-attachments/assets/3e27cffb-fe39-4417-ade4-a81924755f39">
